services/harmony/app/util/edl-api.ts

```diff
@@ -215,3 +215,21 @@ export async function validateUserIsInCoreGroup(
 
   return true;
 }
+
+/**
+ * Returns true if EDL is up
+ *
+ * @param context - request context
+ * @returns A Promise containing `true` if EDL is up
+ */
+export async function isEdlHealthy(context: RequestContext): Promise<boolean> {
```

Comment:
I think it might be better if all of these types of functions were consistent in returning a `Promise<HealthStatus>` instead of `Promise<boolean>` or `Promise<{health: boolean; message: string}>`, etc. And I would name them `getEdlHealth`, `getCmrHealth`, etc.

Reply:
It was sort like what you suggested in the original draft and Chris requested changing cmr health check to be something like the `isEdlHealthy`, and since I also want to preserve the CMR health check message, it became this way.

Reply:
I will rename the function back to `getCmrHealth`.